infra/resources/modules/commons/function_management.tf

```diff
@@ -0,0 +1,352 @@
+    local.management_app_settings,
+    {},
+  )
+
+  sticky_app_setting_names = []
+
+  storage_account_name         = replace(format("%ssubfn01", local.project), "-", "")
+  storage_account_durable_name = replace(format("%ssubfn01", local.project), "-", "")
+
+  storage_account_info = {
+    account_kind                      = "StorageV2"
+    account_tier                      = "Standard"
+    account_replication_type          = "ZRS"
+    access_tier                       = "Hot"
+    advanced_threat_protection_enable = false
+    use_legacy_defender_version       = false
+    public_network_access_enabled     = true
+  }
+
+  internal_storage = {
+    "enable"                     = true,
+    "private_endpoint_subnet_id" = module.pendpoints_snet.id,
+    "private_dns_zone_blob_ids"  = [azurerm_private_dns_zone.privatelink_blob_core.id],
+    "private_dns_zone_queue_ids" = [azurerm_private_dns_zone.privatelink_queue_core.id],
+    "private_dns_zone_table_ids" = [azurerm_private_dns_zone.privatelink_table_core.id],
+    "queues"                     = [],
+    "containers"                 = [],
+    "blobs_retention_days"       = 0,
+  }
+  subnet_id = module.management_snet.id
+
+  allowed_subnets = [
+    module.management_snet.id
+  ]
+
+  # Action groups for alerts
+  action = [
+    {
+      action_group_id    = azurerm_monitor_action_group.error_action_group.id
+      webhook_properties = {}
+    }
+  ]
+
+  tags = var.tags
+}
+
+module "management_fn_staging_slot" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//function_app_slot?ref=v8.9.0"
+
+  name                = "staging"
+  location            = var.location
+  resource_group_name = azurerm_resource_group.management_rg.name
+  function_app_id     = module.management_fn.id
+  app_service_plan_id = module.management_fn.app_service_plan_id
+  health_check_path   = "/info"
+
+  storage_account_name               = module.management_fn.storage_account.name
+  storage_account_access_key         = module.management_fn.storage_account.primary_access_key
+  internal_storage_connection_string = module.management_fn.storage_account_internal_function.primary_connection_string
+
+  node_version                             = "20"
+  always_on                                = "true"
+  runtime_version                          = "~4"
+  application_insights_instrumentation_key = azurerm_application_insights.ai.instrumentation_key
+
+  app_settings = merge(
+    local.management_app_settings,
+    {},
+  )
+
+  subnet_id = module.management_snet.id
+
+  allowed_subnets = [
+    module.management_snet.id
+  ]
+
+  tags = var.tags
+}
+
+resource "azurerm_monitor_autoscale_setting" "function_management" {
+  name                = format("%s-autoscale-01", module.management_fn.name)
+  resource_group_name = azurerm_resource_group.management_rg.name
+  location            = var.location
+  target_resource_id  = module.management_fn.app_service_plan_id
+
+
+  # Scaling strategy
```

Comment:
Does it make sense to use this patterns? I'd keep 1 instance as long as we are not in prod

Reply:
I'd rather keep this config as commented code